rkt/api_service.go

```diff
@@ -735,6 +735,12 @@ func (s *v1AlphaAPIServer) GetLogs(request *v1alpha.GetLogsRequest, server v1alp
 		stage1Path = fmt.Sprintf("/overlay/%s/upper/", stage1TreeStoreID)
 	}
 	path := filepath.Join(getDataDir(), "/pods/run/", request.PodId, stage1Path, "/var/log/journal/")
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		// in case no journal is present, simply send an empty logs response
+		return server.Send(&v1alpha.GetLogsResponse{})
+	}
```

Comment:
To clarify the question: @alban was confused, that we don't send a `io.EOF` ourselves. This is done by the grpc framework. As discussed OOB I'll `return nil` immediately here, there's actually no need to send an empty `GetLogsResponse` object over the wire.